frontend/src/lib/services/app.services.ts

```
@@ -13,7 +16,15 @@ export const initAppPrivateData = async (): Promise<void> => {
   /**
    * If Nns load but Sns load fails it is "fine" to go on because Nns are core features.
    */
-  await Promise.allSettled([Promise.all(initNns), Promise.all(initSns)]);
+  await Promise.allSettled([
+    Promise.all(initNns).then(() =>
+      // When you log in with a new account for the first time, the account is created in the NNS dapp.
+      // If you request imported tokens before the account is created, an `AccountNotFound` error will be thrown.
+      // To avoid this, the imported tokens should only be requested after the NNS accounts have been initialized.
+      Promise.all(get(ENABLE_IMPORT_TOKEN) ? [loadImportedTokens()] : [])
```

What do you think about instead just ignoring the `AccountNotFound` error? If this is a new user, they won't have imported tokens anyway. And then we don't have to make the application slower for existing users.

---

Not sure, this is the first time a user opens the dapp and there is already an error. Let's improve it later if needed.

---

What do you mean by "there is already an error"?
Of course we don't want the user to see an error. If we ignore the error instead of displaying it, then everything should be fine, no?

---

Hmmm… Actually, that’s a great idea. Let me implement it.

---

Done - https://github.com/dfinity/nns-dapp/pull/5344/files#diff-b7553bd5e5ff7d170310160df25eb91964632b5ea4b10c3df8b4fa15e355f4c2R43